### PR TITLE
add role bindings in Kubernetes example

### PR DESCRIPTION
**Pull request by megastef**

### CODE REVIEW

**Comment by otisg** [comment]
Does this need a reviewer?

File: kubernetes/logagent-daemonset.yml [hunk 1]
```diff
@@ -1,20 +1,61 @@
...
+  name: sematext-logagent
+  labels:
+    app: sematext-logagent
+---
+# Daemonset
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: sematext-logagent
+  name: st-logagent
```
Comment (adnanrahic) [inline kubernetes/logagent-daemonset.yml]: Why is this `st-logagent` but it's `sematext-logagent` everywhere else? We should keep the naming consistent. Should we use `sematext-logagent` everywhere?

File: kubernetes/logagent-daemonset.yml [hunk 2]
```diff
@@ -1,20 +1,61 @@
...
 spec:
+  selector:
+   matchLabels:
+     app: st-logagent
   template:
     metadata:
       labels:
-        app: sematext-logagent
+        app: st-logagent
```
Comment (adnanrahic) [inline kubernetes/logagent-daemonset.yml]: Same comment about `st-logagent` vs, `sematext-logagent`. Pick one?

File: kubernetes/logagent-daemonset.yml [hunk 3]
```diff
@@ -1,20 +1,61 @@
...
+# Daemonset
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: sematext-logagent
+  name: st-logagent
 spec:
+  selector:
+   matchLabels:
+     app: st-logagent
```
Comment (adnanrahic) [inline kubernetes/logagent-daemonset.yml]: Same comment about `st-logagent` vs, `sematext-logagent`. Pick one?

File: kubernetes/logagent-daemonset.yml (hunk 1 above)
Comment (megastef) [reply kubernetes/logagent-daemonset.yml]: Sematext UI instructions use `st-logagent` and the Sematext helm chart uses st-logagent as daemonset name. This change makes the naming consistent with other documentations and instructions.   For the role bindings, sematext/agent uses `sematext-agent` and so we use for role-bindings `sematext-logagent`. If you give all objects the same name, how can users differentiate the different objects?  

Anyhow, we use here the same naming convention as we have it in helm charts already.

File: kubernetes/logagent-daemonset.yml (hunk 2 above)
Comment (megastef) [reply kubernetes/logagent-daemonset.yml]: Sematext UI instructions use `st-logagent` and the Sematext helm chart uses st-logagent as daemonset name. This change makes the naming consistent with other documentations and instructions.   For the role bindings, sematext/agent uses `sematext-agent` and so we use for role-bindings `sematext-logagent`. If you give all objects the same name, how can users differentiate the different objects?  

Anyhow, we use here the same naming convention as we have it in helm charts already.

**Pull request merged**